### DISCUSSION TITLE
Login doesn't work properly

### DISCUSSION THREAD

**Discussion by Samantha77** [BlueBrain/nexus#2781] [Q&A]
Hi,

Recently our team is actively working on the BBN integration part, and we meet some issues when deploying the BBN using docker swarm. Once all required docker containers initiated, I followed instruction to invoke below api for creating realm:
![image](https://user-images.githubusercontent.com/23462638/133190566-4cae8812-385b-4752-b922-016a03aa02ee.png)
And after realm created, I am able to find Admin subapp and even I am able to login to the keycloak and get redirect back to the web page. But I couldn't find the user information and token at the top right conner. It still shows **login** 
![image](https://user-images.githubusercontent.com/23462638/133190863-9e77eb01-c2cc-4885-80b7-36708013bf1c.png)
Also based on below screenshot, it seems we already get authenticated successfully by keycloak. But don't know why the BBN doesn't really think we are login correctly. I also attached my keycloak client configuration and docker container screenshot below as the reference. Any suggestion could be helpful and I am very appreciate that! Thanks!!!
![image](https://user-images.githubusercontent.com/23462638/133191021-aabad5ad-ec76-406a-85b7-0e4fbff2a851.png)
![image](https://user-images.githubusercontent.com/23462638/133191220-345131b1-363c-46a6-a2bd-05e2f6727851.png)
![image](https://user-images.githubusercontent.com/23462638/133191620-0feab0a8-e1db-48f3-a5ca-f38422fabde0.png)

**Comment by dhaneshnm** [comment]
Hello,
Is this issue happening in your localhost or in a deployed instance?

**Reply by Samantha77** [reply]
Hi,

We tried to deploy that in server as well. But the issue really is that when we deploy the keycloak instance in different domain with BBN, although the realm is created successfully, it won't appear login information (Both Chrome and Safari) after proper login. For instance: BBN deployed in **https://my-domain-1.com/nexus** and keycloak deployed in **https://my-domain-2.com/auth**
However, if the keycloak is deployed under the same domain, it will not trigger this issue. For example, the BBN is deployed under **https://my-domain.com/nexus** and the keycloak is deployed in **https://my-domain.com/auth**

**Reply by bogdanromanx** [reply]
Hi @Samantha77, did you configure the keycloak client (realm -> Clients -> client -> Settings -> Web Origins) with appropriate web origins?
